src/side-navigation/internal.tsx

```diff
@@ -166,9 +176,10 @@ function Link({ definition, expanded, activeHref, fireFollow }: LinkProps) {
 
 interface SectionProps extends BaseItemComponentProps {
   definition: SideNavigationProps.Section;
+  variant: 'section' | 'section-group' | 'link-group' | 'expandable-link-group' | 'root';
```

Comment:
Would an `enum` make sense here?

Reply:
We are not using typescript enums. Some reasons are highlighted there: https://stackoverflow.com/questions/52393730/typescript-string-literal-union-type-from-enum